### PR TITLE
Adding V3 pool roles

### LINKED ISSUE #1086: add owners for v3 pools
v3 pools have different ownership concept, need to integrate this:

On a pool, natively there can be a:
- SwapFeeManager - Can set swap fee
- PauseManager - Can pause the pool
- PoolCreator - Can set fee splits

### PR DESCRIPTION
**Pull request by gmbronco**
closes #1086

### CODE REVIEW

File: modules/pool/pool.gql [hunk 1]
```diff
@@ -165,7 +165,19 @@ type GqlPoolMinimal {
     """
     The wallet address of the owner of the pool. Pool owners can set certain properties like swapFees or AMP.
     """
-    owner: Bytes
+    owner: Bytes @deprecated(reason: "Use swapFeeManager instead")
+    """
+    Account empowered to set static swap fees for a pool (when 0 on V2 swap fees are immutable, on V3 delegate to governance)
+    """
+    swapFeeManager: Bytes
```
Comment (franzns) [inline modules/pool/pool.gql:172]: not sure why this is bytes, lets make it string

**Comment by franzns** [comment]
I would only change the owners types to string instead of bytes. Not sure why the owner was bytes before. Rest looks good

File: modules/pool/pool.gql (hunk 1 above)
Comment (gmbronco) [reply modules/pool/pool.gql:172]: I guess Bytes naming comes from the contracts and it's meant to represent 0x strings. In TS it's just a String alias like BigDecimal is:
https://github.com/balancer/backend/blob/pool-roles/codegen.ts/#L15-L16
Maybe we should remove the mappings altogether to make it more understandable in the docs?

Comment (franzns) [reply modules/pool/pool.gql:172]: agree, lets get rid of bytes and just use strings

Comment (franzns) [reply modules/pool/pool.gql:172]: just change the owners to strings in the schema for now, leave the mapping as is

**Comment by gmbronco** [comment]
> I would only change the owners types to string instead of bytes. Not sure why the owner was bytes before. Rest looks good

it might be a consistency problem, because all other addresses are bytes as well.

**Comment by franzns** [comment]
Ok. Lets leave it like this then and refactor another time 👍

**Pull request merged**